### PR TITLE
Integration test docker compose readme

### PR DESCRIPTION
**Pull request by agricenko**
Readme updates
Added "docker compose" section

### CODE REVIEW

File: integration-tests/README.md [hunk 1]
```diff
@@ -68,6 +68,25 @@ can either be 8 or 11.
...
+docker-compose.yml - Defining minimal Druid cluster that can be used for non cluster tests.
+    
+    docker-compose -f docker-compose.yml up
+    DRUID_INTEGRATION_TEST_GROUP=query docker-compose -f docker-compose.yml up
+
+docker-compose.override-env.yml - the same configuration as docker-compose.yml + override-env variable that needed to run cloud tests 
+
+    OVERRIDE_ENV=./environment-configs/override-examples/s3 docker-compose -f docker-compose.override-env.yml up
+    
+docker-compose.druid-hadoop.yml - druid hadoop
```
Comment (maytasm) [inline integration-tests/README.md:86]: "druid hadoop" is very vague. Can you change to something like "for starting Apache Hadoop 2.8.5 cluster with the same setup as the Druid tutorial".

File: integration-tests/README.md [hunk 2]
```diff
@@ -68,6 +68,25 @@ can either be 8 or 11.
...
+Docker compose yamls located in "docker" folder
+
+docker-compose.base.yml - Base file that defining all containers for integration test
+
+docker-compose.yml - Defining minimal Druid cluster that can be used for non cluster tests.
+    
+    docker-compose -f docker-compose.yml up
+    DRUID_INTEGRATION_TEST_GROUP=query docker-compose -f docker-compose.yml up
+
+docker-compose.override-env.yml - the same configuration as docker-compose.yml + override-env variable that needed to run cloud tests
```
Comment (maytasm) [inline integration-tests/README.md:82]: Maybe add that override-env variable is for file containing Druid property for override default values or something.

File: integration-tests/README.md [hunk 3]
```diff
@@ -68,6 +68,25 @@ can either be 8 or 11.
...
+## Docker compose
+
+Docker compose yamls located in "docker" folder
+
+docker-compose.base.yml - Base file that defining all containers for integration test
+
+docker-compose.yml - Defining minimal Druid cluster that can be used for non cluster tests.
+    
+    docker-compose -f docker-compose.yml up
+    DRUID_INTEGRATION_TEST_GROUP=query docker-compose -f docker-compose.yml up
```
Comment (maytasm) [inline integration-tests/README.md:80]: Can you add what is the different between line 79 and line 80?

File: integration-tests/README.md [hunk 4]
```diff
@@ -68,6 +68,25 @@ can either be 8 or 11.
 Druid's configuration (using Docker) can be overrided by providing -Doverride.config.path=<PATH_TO_FILE>. 
 The file must contain one property per line, the key must start with `druid_` and the format should be snake case. 
 
+## Docker compose
+
+Docker compose yamls located in "docker" folder
+
+docker-compose.base.yml - Base file that defining all containers for integration test
```
Comment (maytasm) [inline integration-tests/README.md:75]: defining -> defines

File: integration-tests/README.md [hunk 5]
```diff
@@ -68,6 +68,25 @@ can either be 8 or 11.
...
+
+docker-compose.yml - Defining minimal Druid cluster that can be used for non cluster tests.
```
Comment (maytasm) [inline integration-tests/README.md:77]: defining -> defines

Comment (agricenko) [reply integration-tests/README.md]: Fixed

File: integration-tests/README.md (hunk 4 above)
Comment (agricenko) [reply integration-tests/README.md]: Fixed

File: integration-tests/README.md (hunk 2 above)
Comment (agricenko) [reply integration-tests/README.md:83]: Added some details

File: integration-tests/README.md (hunk 1 above)
Comment (agricenko) [reply integration-tests/README.md]: fixed

File: integration-tests/README.md (hunk 3 above)
Comment (agricenko) [reply integration-tests/README.md]: Added some details.
2nd docker-compose need to be used for security/query tests. Druid docker container update keys in druid.sh if DRUID_INTEGRATION_TEST_GROUP="query" or "security"
druid/integration-tests/docker/druid.sh:85

**Pull request merged**